dashboard/app/views/pd/application/teacher1819_application_mailer/local_summer_accepted.html.haml

```diff
@@ -90,6 +90,6 @@
   %br/
   Code.org
 %p
-  = @application.regional_partner.contact.name
+  = @application.regional_partner.program_managers.first.name
```

Comment:
Question of style: if we're ultimately going to remove `contact_id` and make contact `contact` return the first program manager, then it might make sense (more logic in the model) to continue calling `contact` here. However if we plan on removing `contact` entirely and making the consumer decide what to use then this makes sense.

Reply:
Oh good point. I don't feel strongly about one approach vs the other, but it would be easier to take the first approach and not have to worry about replacing everywhere `contact` is used.